### PR TITLE
Spe 319 ocpp simulator getsetvar support part4

### PR DESCRIPTION
**Pull request by RomanVlasenko**
- Reduced number of arguments in some methods by introducing AttributePath object
- Also increased number of test cases to cover all attributes of each variable

### CODE REVIEW

File: simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/component/variable/attribute/AttributePath.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import lombok.Builder;
+import lombok.Value;
+
+/**
+ * Contains all data necessary to access arbitrary variable's attribute of the station
+ */
+@Value
+@Builder
+@AllArgsConstructor
+public class AttributePath {
```
Comment (draganevbox) [inline simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/component/variable/attribute/AttributePath.java]: Should it be named AttributeProperties or something like that? Why AttributePath?

Comment (RomanVlasenko) [reply simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/component/variable/attribute/AttributePath.java]: Cause it represents a path to the attribute.
Similar to directory path, but in this case: `componentName\componentInstance\evseId\connectorId\variableName\variableInstance\attributeType`

Comment (draganevbox) [reply simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/component/variable/attribute/AttributePath.java]: Could we put that in the class' javadoc?

Comment (RomanVlasenko) [reply simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/component/variable/attribute/AttributePath.java]: it's there, just in a different phrasing

Comment (draganevbox) [reply simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/component/variable/attribute/AttributePath.java]: Okay... where can I find this in the source code?
componentName\componentInstance\evseId\connectorId\variableName\variableInstance\attributeType

Comment (RomanVlasenko) [reply simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/component/variable/attribute/AttributePath.java]: there's component, which contains component name, component instance and evse and connector id
variable which contains name and instance
attribute type

File: simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/component/exception/IllegalAttributeTypeException.java [hunk 2]
```diff
@@ -0,0 +1,9 @@
+package com.evbox.everon.ocpp.simulator.station.component.exception;
+
+public class IllegalAttributeTypeException extends RuntimeException {
+
+    public IllegalAttributeTypeException(String value) {
+        super("Unknown attribute type: " + value);
```
Comment (aleksandr-nesterov) [inline simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/component/exception/IllegalAttributeTypeException.java]: Please dont put additional logic to constructors

Comment (RomanVlasenko) [reply simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/component/exception/IllegalAttributeTypeException.java]: ok, will fix

**Pull request merged**